docs/volume/create-volume.md

```diff
@@ -29,6 +29,12 @@ description: Create a volume from the Volume page.
 
 ![create-empty-volume](/img/v1.2/volume/create-empty-volume.png)
 
+:::info important
+
+Harvester attaches/detaches volumes based on user operations dynamically. If you plans to attach any volume to certain node manually, it may block the [node maintenance](../host/host.md#node-maintenance), check the affection of [Manually Attached Volumes](../troubleshooting/host.md#manually-attached-volumes) and set a proper global option.
```

Comment:
```suggestion
Harvester attaches/detaches volumes based on user operations dynamically. If you plans to attach any volume to certain node manually, it may block the [node maintenance](../host/host.md#node-maintenance), check the section [Manually Attached Volumes](../troubleshooting/host.md#manually-attached-volumes) and set a proper global option.
```